scipy/sparse/linalg/_eigen/arpack/tests/test_arpack.py

```diff
@@ -396,7 +398,10 @@ def __init__(self):
         self.complex_test_cases = [SNC, GNC]
 
 
-def test_symmetric_modes():
+@pytest.mark.iterations(1)
+@pytest.mark.thread_unsafe
+def test_symmetric_modes(num_parallel_threads):
+    assert num_parallel_threads == 1
```

Comment:
Making triple-sure this doesn't run more than once - was the effect that bad?

Reply:
I was checking if somehow the number of threads was being changed somehow, but it was changing because of the number of iterations and this particular or operation in `pytest-run-parallel`: https://github.com/Quansight-Labs/pytest-run-parallel/blob/76a04a29a269db3ea6c81b4151c1f3903bde3ae8/src/pytest_run_parallel/plugin.py#L116